modules/lib/lib-project/src/main/resources/lib/xp/project.js

```diff
@@ -0,0 +1,75 @@
+exports.create = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.project.CreateProjectHandler');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5502328)